### PR TITLE
[13.0][FIX] account_reconcile_payment_order: Fix tests

### PR DESCRIPTION
**Pull request by joao-p-marques**
Create journal instead of searching
Use computed value for total price instead of hardcoded (helps avoid integration tests when another module changes fiscal plan)

@Tecnativa
TT28962

ping @victoralmau @pedrobaeza

### CODE REVIEW

**Review by victoralmau — APPROVED** [review]

File: account_reconcile_payment_order/tests/test_account_reconcile_payment_order.py [hunk 1]
```diff
@@ -31,17 +25,29 @@ def setUpClass(cls):
...
                 "journal_id": cls.bank_journal.id,
                 "line_ids": [
-                    (0, 0, {"date": "2019-01-01", "name": "Test line", "amount": 230}),
+                    (
+                        0,
+                        0,
+                        {
+                            "date": "2019-01-01",
+                            "name": "Test line",
+                            "amount": payment_order.total_company_currency,
```
Comment (pedrobaeza) [inline account_reconcile_payment_order/tests/test_account_reconcile_payment_order.py]: This goes against the nature of the tests. We should fix data.

Comment (joao-p-marques) [reply account_reconcile_payment_order/tests/test_account_reconcile_payment_order.py]: Ok, I think it should be good now, along with https://github.com/OCA/bank-payment/pull/797 (this tests use the base from that module)

Comment (victoralmau) [reply account_reconcile_payment_order/tests/test_account_reconcile_payment_order.py]: PR it's complete now, it's ready to merge?

Comment (joao-p-marques) [reply account_reconcile_payment_order/tests/test_account_reconcile_payment_order.py]: It should be completed with the latest change, although test here will only pass after https://github.com/OCA/bank-payment/pull/798 is merged

**Pull request merged**